Documentation/project-docs/developer-guide.md

```diff
@@ -71,7 +76,9 @@ Commands added as targets can be invoked once the target project adds a referenc
 Targets are invoked by calling `dotnet msbuild /t:{TargetName}`
 
 ### Commands on the PATH
+
 The dotnet CLI considers any executable on the path named `dotnet-{commandName}` to be a command it can call out to. 
 
 ## Things to Know
+
 - Any added commands are usually invoked through `dotnet {command}`. As a result of this, stdout and stderr are redirected through the driver (`dotnet`) and buffered by line. As a result of this, child commands should use Console.WriteLine in any cases where they expect output to be written immediately. Any uses of Console.Write should be followed by Console.WriteLine to ensure the output is written.
```

Comment:
Completely unrelated, but this is the first I read of this and it's terrifying. Why are we redirecting and buffering output in this case?

Reply:
I imagine this question is for @livarcocc right @nguerrera?

Reply:
Correct. :)